### DISCUSSION TITLE
Translations - Sort/Avoid

### DISCUSSION THREAD

**Discussion by bluehenve** [DaxStudio/DaxStudio#1144] [Q&A]
Hi

I see translated names in the explorer section on the left side of Dax Studio:

- The translated names are sorted by model names ... how can I sort by translation names?
- Is it possible to show model names instead of translated names?

I run Dax Studio 3.0.8
regards, Henrik

**Comment by dgosbell** [comment]
I don't currently sort by the translation name, but I think I should see if I can make this the default. We also don't have the option to show the model names instead of the translated names, but I think that is an excellent idea to add as an option.

**Reply by dgosbell** [reply]
I've added 2 issues to track these requests:
* #1146 
* #1147

**Reply by bluehenve** [reply]
Suggestion: 
   Instead of an option ... show a translation dropdown including 'No translation' in the Metadata pane, se picture.
Reason: 
   Even when using translation you often need a view of the model name anyhow, as DAX is using model names

![Clipboard](https://github.com/DaxStudio/DaxStudio/assets/70111241/89860276-67b2-4e68-8669-a43fb32046e2)

**Reply by dgosbell** [reply]
You can already choose which language you want in the Advanced Properties of the connection dialog and we cannot change the translation without re-connecting. I'm not sure if this is a common enough use case for it to be always visible in the metadata browser. 

I could add a right click option to switch the view between the translation or the model names. 

The intellisense and drag/drop should already always use the model names.

**Reply by bluehenve** [reply]
"I could add a right click option to switch the view between the translation or the model names."
That would be a wonderfull solution, regards Henrik

**Comment by bluehenve** [comment]
Fantastic - this will be a huge relief, as working with translation is a pain in the first place 👍💪
